lib/order.rb

```diff
@@ -0,0 +1,7 @@
+class Order
+  attr_accessor :id, :customer_name, :date_placed, :items
```

Comment:
You should avoid using `attr_accessor`s where possible. Use `attr_reader`s if you have to, and keep fields private where possible.